<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseColorDialog</class>
 <widget class="QDialog" name="ChooseColorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnHearts">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>40</y>
     <width>75</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDiamonds">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>75</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClubs">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>40</y>
     <width>75</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSpades">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>75</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lblHeadline">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>What color do you wish?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
